app/models/game.rb

```diff
@@ -0,0 +1,12 @@
+class Game < ActiveRecord::Base
+  validates :name, presence: true
+  validates :white_player_id, presence: true
+  validates :black_player_id, presence: true
```

Comment:
Good point. I'll replace these with a custom validation that looks for at least one ID and will remove the constraints on the columns.